packages/@guardian/source-foundations/src/index.ts

```diff
@@ -98,7 +98,6 @@ export type {
 	FontWeightDefinition,
 	Option,
 	TypographyStyles,
-	TypographySizes,
```

Comment:
This would be a breaking change. Is there a way to keep it in for now? I don't think anyone is using it yet, so maybe we can mark it as deprecated and remove it in the next major version

Reply:
fixed in [5925e0f](https://github.com/guardian/source/pull/1202/commits/5925e0ffc03013e9af5877d4f422b91ffbe24597)